flutter/lib/ui/geometry.dart

```diff
@@ -837,39 +982,47 @@ class RRect {
   }
 
   /// The point halfway between the left and right and the top and bottom edges of this rectangle.
-  Point get center => new Point(left + width / 2.0, top + height / 2.0);
+  Point get center => new Point(left + width / 2, top + height / 2);
```

Comment:
`left` and `width` are already `double`, so I was just trying to keep everything more consistent.